source/site-administrators/engine/engine-site-security-guide.rst

```diff
@@ -117,322 +149,33 @@ be used:
-          <label for="password">Password: </label>
-          <input type="password" name="password"/>
-          <br/>
-          <input type="checkbox" name="rememberMe" value="true">Remember Me</input>
-          <br/>
-          <button type="submit">Sign in</button>
-      </form>
-
-Add Logout
-==========
-
-To add logout, just add a link in the global header that points to /crafter-security-logout:
-
-.. code-block:: html
-  :linenos:
-
-  <a href="/crafter-security-logout">Log Out</a>
-
-Add Single Sign-On
-==================
-
-Configure SSO headers with at least a MELLON_secure_key, MELLON_username, MELLON_email and MELLON_groups (which must be a comma separated list of string) in the header, then check in Crafter Profile Admin Console to make sure
-that the Single sign-on enabled checkbox is selected in the tenant page.
-
-.. image:: /_static/images/sso_enabled.png
-
-All headers with the ``MELLON_`` prefix will be mapped, without the prefix, to the attributes you defined in the 
-Crafter Profile tenant, when a new user needs to be created. So the configuration above will cause the Security 
-Provider to create a user with firstName, lastName and displayName attributes.
-
-Add Facebook Login
-==================
-
-#.  Be sure there's a connections attribute of Complex type defined for the site's Crafter Profile Tenant. This 
-    attribute is needed to store the Facebook connection info. To add this attribute to the Tenant, go to Crafter 
-    Profile Admin Console, select the Tenant and then add the attribute.
-
-    .. image:: /_static/images/connections_attribute.png
-
-#.  Add the Facebook appSecret and appKey to your site's config (in Studio, Config > Engine Site Configuration), like this:
-
-    .. code-block:: xml
-      :linenos:
-
-      <socialConnections>
-          <facebookConnectionFactory>
-              <appId>000000000000000</appId>
-              <appSecret>c852cb30cda311e488300800200c9a66</appSecret>
-          </facebookConnectionFactory>
-      </socialConnections>
-
-#.  Add a JS method that is triggered when the user clicks on the "Login with Facebook" button, that displays the FB 
-    login popup when the user clicks on "Connect with Facebook":
-
-    .. code-block:: javascript
-      :linenos:
-
-      $("#connect").click(function() {
-          try {
-              var top = (screen.height / 2) - (300/ 2);
-              var left = (screen.width / 2) - (500 / 2);
-              var fbDialog = window.open('/connect/facebook_dialog', 'fbDialog', 'width=500, height=300, top=' + top + ', left=' + left);
-              var interval = setInterval(function() {
-                  if (fbDialog == null || fbDialog.closed) {
-                      clearInterval(interval);
-
-                      location.reload();
-                  }
-              }, 1000);
-          } catch(e) {}
-      }
+Once the authentication and authorization configurations are completed you can use the ``authToken`` object in
```

Comment:
Isn't the base interface of `authToken` the Spring `Authentication` interface? Maybe we can mention that and link that Javadoc.

Reply:
That's already documented in the freemarker/groovy variables, let me know if we should add a link to that or to the class directly

Reply:
Let's do that to make it more clear